cli/tests/unit/spawn_test.ts

```diff
@@ -248,6 +248,27 @@ Deno.test(
   },
 );
 
+Deno.test(
+  { permissions: { run: true, read: true } },
+  async function spawnAbort() {
+    const ac = new AbortController();
+    const child = Deno.spawnChild(Deno.execPath(), {
+      args: [
+        "eval",
+        "setTimeout(console.log, 1e8)",
+      ],
+      signal: ac.signal,
+      stdout: "null",
+      stderr: "null",
+    });
+    queueMicrotask(() => ac.abort());
+    const status = await child.status;
+    assertEquals(status.success, false);
+    assertEquals(status.code, 143);
+    assertEquals(status.signal, 15);
```

Comment:
Windows doesn't seem to use special exit codes based on the signal sent, but will just have a non-zero exit code (seems to be application specific).

One of the kill tests does the following:

```suggestion
  if (Deno.build.os === "windows") {
    assertEquals(status.code, 1);
    assertEquals(status.signal, undefined);
  } else {
    assertEquals(status.success, false);
    assertEquals(status.code, 143);
  }
```